### ISSUE TITLE
github.com account is not properly detected

### ISSUE THREAD

**Issue by wiktor-k** [aaronpk/IndieAuth.com#176]
Hi,
I'm trying to use IndieAuth.com but the Github provider throws an error and it's not clear why:

```
Unknown error: undefined method `value` for nil:NilClass
```

![github](https://user-images.githubusercontent.com/1718963/35476221-8ea15e96-03ac-11e8-91cd-0f79e896fe58.png)

This can be reproduced by using https://metacode.biz/@wiktor at https://indieauth.com/ Try it! box.

**Comment by wiktor-k** [comment]
Okay I've found what's the root cause, I had links (`<a>`) that did not have `href`s. Targets were updated via JavaScript (to avoid mail harvesting bots).

Maybe it would be useful to add checks in `lib/relparser.rb` for presence of `href` before de-referencing it via `.value` and throw appropriate error? So that the person configuring own site can see the problem, or at least document the current behavior.
